assets/src/features/shared/components/MembraneLogo.tsx

```diff
@@ -12,53 +12,98 @@ const MembraneLogo: React.FC<React.SVGAttributes<SVGElement>> = (props) => {
-          fill="#001A72"
-        />
-        <path
-          d="M142.209 35.0531C146.504 35.0531 149.191 31.9326 149.191 26.8968C149.191 22.2015 146.565 19.2968 142.178 19.2968C139.15 19.2968 137.05 20.7175 136.401 22.9427H136.37C136.401 22.2015 136.432 20.9657 136.432 20.3169V13.3965H132.973V34.8066H136.432C136.37 34.0346 136.309 32.6446 136.309 31.5629H136.34C136.957 33.694 139.058 35.0531 142.209 35.0531V35.0531ZM141.314 32.2116C138.317 32.2116 136.432 30.5735 136.432 27.9169V27.1449C136.432 24.0553 138.347 22.1399 141.375 22.1399C144.094 22.1399 145.732 23.9937 145.732 26.9909C145.732 30.2346 144.032 32.2116 141.314 32.2116Z"
-          fill="#001A72"
-        />
-        <path
-          d="M152.804 34.8063H156.264V28.134C156.264 24.4881 157.902 22.2646 160.528 22.2646C162.566 22.2646 163.556 23.4696 163.556 26.0337C163.556 26.7441 163.494 27.5161 163.432 28.1032H166.584C166.645 27.4545 166.738 26.2186 166.738 25.2617C166.738 21.4617 164.915 19.2998 161.609 19.2998C158.304 19.2998 156.883 21.1844 156.171 23.9334H156.141L156.264 19.5463H152.804V34.808V34.8063Z"
-          fill="#001A72"
-        />
-        <path
-          d="M175.694 35.0529C179.061 35.0529 180.884 33.6937 181.378 31.4377H181.409C181.409 32.6736 181.317 34.0635 181.255 34.8047H184.715V25.505C184.715 21.4893 181.996 19.2949 177.361 19.2949C173.005 19.2949 170.348 21.1179 170.348 24.1767C170.348 24.454 170.379 24.8563 170.41 25.072H173.592C173.561 24.8254 173.561 24.5773 173.561 24.4849C173.561 22.7852 174.889 21.8899 177.238 21.8899C179.925 21.8899 181.253 22.91 181.253 25.1952V26.277H175.6C171.521 26.277 169.884 28.1 169.884 30.416C169.884 33.3823 172.202 35.0496 175.692 35.0496L175.694 35.0529ZM176.404 32.4579C174.302 32.4579 173.068 31.6551 173.068 30.2651C173.068 29.0601 173.932 28.3805 175.911 28.3805H181.255V28.6579C181.255 31.0063 179.401 32.4579 176.404 32.4579Z"
-          fill="#001A72"
-        />
-        <path
-          d="M188.644 34.8065H192.104V27.4222C192.104 24.0861 194.019 22.2631 197.047 22.2631C199.796 22.2631 201.155 23.6222 201.155 26.0323V34.8065H204.615V25.2603C204.615 21.4911 202.235 19.2983 198.374 19.2983C194.82 19.2983 192.843 20.8732 192.01 23.6238H191.979L192.102 19.5465H188.643V34.8081L188.644 34.8065Z"
-          fill="#001A72"
-        />
-        <path
-          d="M223 26.6194C223 21.8625 220.251 19.2983 215.925 19.2983C211.105 19.2983 208.079 22.3572 208.079 27.3006C208.079 32.244 211.107 35.0546 215.925 35.0546C220.097 35.0546 222.877 32.9852 223 29.5858H219.879C219.725 31.3779 218.211 32.3656 215.987 32.3656C213.392 32.3656 211.754 30.7908 211.569 27.8552H223V26.6194ZM219.879 25.7549H211.662C212.064 23.2216 213.577 21.9858 215.956 21.9858C218.459 21.9858 219.879 23.2524 219.879 25.5392V25.7549Z"
-          fill="#001A72"
-        />
-      </g>
-      <defs>
-        <clipPath id="clip0_80_14420">
-          <rect width="223" height="49" fill="white" />
-        </clipPath>
-      </defs>
+      <path
```

Comment:
It seems the logo on the join screen and inside the room differ. I know this matches the designs, but was that intended?

Reply:
It is designed this way. https://www.figma.com/file/lVPaILYT73AJIOsoXNbsXj?node-id=15:5670#365690611